tests/services/order_validation_service/sample_rules/conftest.py

```diff
@@ -114,3 +117,35 @@ def valid_fastq_order() -> FastqOrder:
         samples=samples,
         name="FastqOrder",
     )
+
+
+def create_rml_sample(id: int) -> RmlSample:
+    return RmlSample(
+        application="WGSPCFC030",
```

Comment:
Fairly certain that this app tag is not valid for RML but we can fix that in a future PR where we add an integration test